### PR TITLE
add farsi language

### PR DESCRIPTION
**Pull request by Ali-Rnik**
I tried to add Farsi language files with the help of google translate with the following script:
https://gitlab.com/ali_rnik/auto_i18n_ospos

### CODE REVIEW

File: application/helpers/locale_helper.php [hunk 1]
```diff
@@ -67,6 +67,7 @@ function get_languages()
 		'es:spanish' => 'Spanish',
 		'es-MX:spanish' => 'Spanish (Mexico)',
 		'fr:french' => 'French',
+		'fa-IR:persian' => 'Farsi (Iran'),
```
Comment (daN4cat) [inline application/helpers/locale_helper.php:70]: There is a mistake here. The single quote ' should go after the ).

Comment (Strongsaxophone) [reply application/helpers/locale_helper.php]: What a bad mistake, now it has been fixed and I hope it pass the integration.

**Comment by Ali-Rnik** [comment]
in the the Demo, After I changed locale to "Farsi ( Iran )"  and then moved to Home menu, only a blank page is loaded.

**Comment by objecttothis** [comment]
> 
> 
> in the the Demo, After I changed locale to "Farsi ( Iran )" and then moved to Home menu, only a blank page is loaded.
> what is the possible issue of this commit?

You'll need to look at the error log.  This can't be done on the demo, so you'll need to download this build in bintray (https://bintray.com/jekkos/opensourcepos/opensourcepos/view/files?sort=updated&order=desc#files) and install it on your server.  Enable the logs in the config.php.  When you get the blank screen you should be able to go to /application/logs/.  That log will give you a clue as to what is going on.  Is the language setup in weblate also?

**Pull request merged**